### PR TITLE
twin console app

### PR DESCRIPTION
**Pull request by norelina**
- Updated Abode console app to upload models, create twins and a relationship.
  - Abode console app requires ADT Host name with https:// prefix to be set in appsettings.json.
- Added Abode.Tests project.

### CODE REVIEW

**Comment by charleszipp** [comment]
@norelina instead of having two apps here, I would suggest you overwrite what existed in the projects (Abode.CLI/Domain, etc) with your examples as they are much more complete. Same for they models. Feel free to replace room and floor with the ones from your example. This way we end up with one console app in the example.

File: src/TestAbode/TestAbode.csproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (charleszipp) [inline src/TestAbode/TestAbode.csproj]: To be consistent with the names of other projects, lets rename this to Abode.Tests

File: src/Abode.CLI/IDigitalTwinsClient.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using Azure;
```
Comment (charleszipp) [inline src/Abode.CLI/IDigitalTwinsClient.cs:1]: Dont change anything with this PR but, something to think about for future. In an e2e twins solution, there would likely be multiple services needing to use the twins client (i.e. service for ingress (creating twins), one for reads/queries, one for managing models, potentially a CLI for internal ops like we have here. What might you do to facilitate reuse of this abstraction between all these services?

**Review by charleszipp — APPROVED** [review]

File: src/TestAbode/TestAbode.csproj (hunk 1 above)
Comment (norelina) [reply src/TestAbode/TestAbode.csproj]: Renamed. Thank you.

**Pull request merged**